src/commands/createFunction.ts

```diff
@@ -49,8 +51,9 @@ async function validateIsFunctionApp(outputChannel: vscode.OutputChannel, functi
     }
 }
 
-async function promptForFunctionName(ui: IUserInterface, functionAppPath: string, template: Template): Promise<string> {
-    const defaultFunctionName: string | undefined = await fsUtil.getUniqueFsPath(functionAppPath, template.defaultFunctionName);
+async function promptForFunctionName(ui: IUserInterface, functionAppPath: string, template: Template, language: string): Promise<string> {
```

Comment:
You'll have to update `validateFunctionName` for Java functions as well

Reply:
Do you mean ```validateTemplateName()```?

Reply:
Yes. `fsUtil.getUniqueFsPath` also doesn't apply in the Java case

Reply:
I think ```fsUtil.getUniqueFsPath``` is OK for Java case